src/storage/mutate/AddEdgesProcessor.cpp

```diff
@@ -267,8 +268,9 @@ void AddEdgesProcessor::doProcessWithIndex(const cpp2::AddEdgesRequest& req) {
             continue;
         }
         env_->kvstore_->asyncAppendBatch(spaceId_, partId, std::move(batch),
-            [l = std::move(lg), partId, this](kvstore::ResultCode kvRet) {
+            [l = std::move(lg), &wrapper, partId, this](kvstore::ResultCode kvRet) {
```

Comment:
You can't use `&wrapper` here, it is basically like previous before. Because `wrapper` will be released once it is out of scope. Perhaps you need to add a move constructor of `IndexCountWrapper`, just like the `lg` here.

Reply:
Yeah， 👍